Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM ruby:3.2.2-alpine
 ARG BUNDLE_INSTALL_CMD
-ENV RACK_ENV=development
```

Comment:
I think this should still be RACK_ENV. The DEPLOY_ENV is supposed to refer to the AWS environment (staging, production, alpaca etc) and RACK_ENV to the environment RAILS runs in (test, development, production). This line tells the container to run in Rails in development mode.